server/models/migrations/v0.8.0-v0.9.0/migrate.sql

```diff
@@ -837,3 +837,10 @@ BEGIN
     UPDATE voucher SET reversed = 0 where voucher.uuid = uuid;
   END IF;
 END $$
+
+
+
+CREATE TABLE `tags`(
+  `uuid` BINARY(16) NOT NULL PRIMARY KEY,
+  `name` VARCHAR(100) NOT NULL
```

Comment:
In the schema, the `name` is unique, but in the migrate script, you do not have `UNIQUE`. You also don't have a test for uniqueness.  Which should it be?